modules/efk-logging-fluentd-external.adoc

```diff
@@ -51,10 +76,45 @@ $ oc edit configmap/fluentd -n openshift-logging
 ----
 <1> Enter the name, host, and port for your external Fluentd server.
 
+. Add the path to your certificate and private key to the `secure-forward.conf` section:
++
+[source,yaml]
+----
+  secure-forward.conf: |
+    # <store>
+    # @type secure_forward
+
+    # self_hostname ${hostname}
+    # shared_key <SECRET_STRING>
+
+    secure yes
+    # enable_strict_verification yes
+
+    ca_cert_path /etc/fluent/keys/ca.crt <1>
+    ca_private_key_path /etc/fluent/keys/privkey.pem <2>
+      # for private CA secret key
+    # ca_private_key_passphrase passphrase
+
+    # <server>
+      # or IP
+    #   host server.fqdn.example.com
+    #   port 24284
+    # </server>
+    # <server>
+      # ip address to connect
+    #   host 203.0.113.8
+      # specify hostlabel for FQDN verification if ipaddress is used for host
+    #   hostlabel server.fqdn.example.com
+    # </server>
+    # </store>
+----
+<1> Add the path to your certificate.
+<2> Add the path to your private key.
```

Comment:
probably omit this and say something like "If you want to use mTLS (client cert auth), see the https://docs.fluentd.org/output/forward page for information about client cert and key parameters and other settings.".